camera:
  surrounding_front:
    sensorID: IMX390
    ispID: GW5200
    resolution: 1280 x 720
    fps: 20
    imageFormat: UYVY
    fieldOfView: 190
    sn: H190X-G03150621
    model: fisheye
    imageWidth: 1280
    imageHeight: 720
    fx: 311.5319594126301
    fy: 310.684697261162
    cx: 643.6709427383354
    cy: 344.96681732416835
    K1: 0.06494872859950979
    K2: 0.0019246522662558574
    K3: -0.012888995393555109
    K4: 0.002060082123840094
    K5: 0.0000000000
    K6: 0.0000000000
    P1: 0.0000000000
    P2: 0.0000000000
  surrounding_left:
    sensorID: IMX390
    ispID: GW5200
    resolution: 1280 x 720
    fps: 20
    imageFormat: UYVY
    fieldOfView: 190
    sn: H190X-G03150594
    model: fisheye
    imageWidth: 1280
    imageHeight: 720
    fx: 309.75404246387063
    fy: 308.7062561238783
    cx: 640.2579529361784 
    cy: 346.09928566542095
    K1: 0.06494872859950979
    K2: 0.0019246522662558574
    K3: -0.012888995393555109
    K4: 0.002060082123840094
    K5: 0.0000000000
    K6: 0.0000000000
    P1: 0.0000000000
    P2: 0.0000000000
  surrounding_right:
    sensorID: IMX390
    ispID: GW5200
    resolution: 1280 x 720
    fps: 20
    imageFormat: UYVY
    fieldOfView: 190
    sn: H190X-G03150627
    model: fisheye
    imageWidth: 1280
    imageHeight: 720
    fx: 311.3353810989308
    fy: 310.27136303145244
    cx: 640.4092225909537
    cy: 345.75067446045995
    K1: 0.06494872859950979
    K2: 0.0019246522662558574
    K3: -0.012888995393555109
    K4: 0.002060082123840094
    K5: 0.0000000000
    K6: 0.0000000000
    P1: 0.0000000000
    P2: 0.0000000000
  surrounding_back:
    sensorID: IMX390
    ispID: GW5200
    resolution: 1280 x 720
    fps: 20
    imageFormat: UYVY
    fieldOfView: 190
    sn: H190X-G03150581
    model: fisheye
    imageWidth: 1280
    imageHeight: 720
    fx: 310.584472434853
    fy: 309.5368604489676
    cx: 640.2405941320368
    cy: 343.7103106024731
    K1: 0.06494872859950979
    K2: 0.0019246522662558574
    K3: -0.012888995393555109
    K4: 0.002060082123840094
    K5: 0.0000000000
    K6: 0.0000000000
    P1: 0.0000000000
    P2: 0.0000000000
